.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Reviewers
+<!--- Call out additional reviewers -->
+<!--- in addition to the assigned reviewer -->
+<!--- example: @heyYou -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Related Issue
+<!--- This project only accepts pull requests related to open issues -->
+<!--- If suggesting a new feature or change, please discuss it in an issue first -->
+<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
+<!--- Please link to the issue here: -->
+
+## Closes
+<!--- You can use "closes #42" to automatically -->
+<!--- close an issue ticket when the -->
+<!--- pull request is merged -->
```

Comment:
you could probably just put these comments in the related issues section along with an uncommented "Closes: #" line to avoid redundant references to issues.